strings: negative Repeat count

I keep running into the following error when using sangrenel (which is awesome btw), and figured since the source of the problem is in this repo I would leave it here:
```
panic: strings: negative Repeat count

goroutine 1 [running]:
strings.Repeat(0x710c99, 0x1, 0x8000000000000000, 0x0, 0xc420937c80)
        /home/strudle/.gvm/gos/go1.8/src/strings/strings.go:432 +0x258
github.com/jamiealquiza/tachymeter.(*Histogram).String(0xc421362c40, 0x32, 0xc420fdbf28, 0x5)
        /home/strudle/.gvm/pkgsets/go1.8/global/src/github.com/jamiealquiza/tachymeter/tachymeter.go:271 +0x2a6
main.main()
        /home/strudle/.gvm/pkgsets/go1.8/global/src/github.com/jamiealquiza/sangrenel/sangrenel.go:161 +0xd72
```

I am able to reproduce this error every time with this snippet:

```
hist := tachymeter.Histogram{map[string]uint64{
        "0-10 10s": uint64(10),
    }}
fmt.Println(hist.String(10)) // throws negative string error
```

This happens because the `scale` function returns a negative value, but a multitude of different inputs can cause a negative return value so I am not sure exactly what is causing it.

I haven't had a whole lot of time to look into this, and you may have already fixed it but I thought I would post anyways.

--- a/tachymeter.go
+++ b/tachymeter.go
@@ -261,6 +261,15 @@ func (h *Histogram) String(s int) string {
 		}
 	}
 
+	// Handle cases of no or
+	// a single bin.
+	switch len(*h) {
+	case 0:
+		return ""
+	case 1:
+		min = 0
+	}
+
 	var b bytes.Buffer
 
 	// Build histogram string.
@@ -278,6 +287,6 @@ func (h *Histogram) String(s int) string {
 
 // Scale scales the input x with the input-min a0,
 // input-max a1, output-min b0, and output-max b1.
-func scale(x float64, a0, a1, b0, b1 float64) float64 {
+func scale(x, a0, a1, b0, b1 float64) float64 {
 	return (x-a0)/(a1-a0)*(b1-b0) + b0
 }

